agent/stats/common_test.go

```diff
@@ -29,6 +29,9 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/tcs/model/ecstcs"
 
 	"github.com/aws/aws-sdk-go/aws"
+	containerSDK "github.com/docker/docker/api/types/container"
```

Comment:
How about calling this `dockercontainer` or `containertype` instead of `containerSDK`?